src/main/routes/task-list.ts

```diff
@@ -0,0 +1,19 @@
+import * as express from 'express'
+import { SubTaskListBuilder } from '../views/macros/elements/task-list/util/subTaskListBuilder';
+
+const router = express.Router()
```

Comment:
We will try to make all route files follow same pattern but for now this is okay